docs/manuals/core/config/index.md

```diff
@@ -35,10 +34,8 @@ organized as follows:
   the `DataNodeConfig^` class.
 - [Task configs](task-config.md) section describes the tasks' configurations using the
   `TaskConfig^` class.
-- [Pipeline configs](pipeline-config.md) section describes the pipelines' configurations using the
-  `PipelineConfig^` class.
 - [Scenario configs](scenario-config.md) section describes the scenarios' configurations using the
-  `ScenarioConfig^` class.
+  `ScenarioConfig^` class. It also describes how to configure a sequence by using the `ScenarioConfig^`.
```

Comment:
It s not blocking for me, but I would not even mention the sequence here. It s part of the scenario configuration.

```suggestion
  `ScenarioConfig^` class.
```